utils/release/git-publish-all.mjs

```diff
@@ -102,6 +133,77 @@ const getCliChangelog = () => {
   });
 })();
 
+async function commitChanges(
```

Comment:
There's a lot of utilities in this file, I think it may be a good idea to extract this one into a separate file. I really don't feel strongly about it though.

Reply:
👎 I think the release scripts should be as much be of possible be scripts.

If we were doing manual releases, we would have a list of successive steps a human would do.
We have code that does it instead of a human, but I think it is best for the code to have the same structure as the 'instructions set' we would give to a human.